doc/DeveloperGuide.md

```diff
@@ -94,6 +94,8 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should not require more than 1 Mb of memory to run.
+6. Access to data should be within 1 second. 
```

Comment:
Can you provide more details regarding the program load or amount of maximum storage size?